src/components/SecondaryPanes/CommandBar.js

```diff
@@ -244,23 +258,8 @@ class CommandBar extends Component {
   }
 }
 
-CommandBar.propTypes = {
-  sources: PropTypes.object,
-  selectedSource: PropTypes.object,
-  resume: PropTypes.func,
-  stepIn: PropTypes.func,
-  stepOut: PropTypes.func,
-  stepOver: PropTypes.func,
-  breakOnNext: PropTypes.func,
-  pause: ImPropTypes.map,
-  pauseOnExceptions: PropTypes.func,
-  shouldPauseOnExceptions: PropTypes.bool,
-  shouldIgnoreCaughtExceptions: PropTypes.bool,
-  isWaitingOnBreak: PropTypes.bool
-};
-
 CommandBar.contextTypes = {
-  shortcuts: PropTypes.object
+  shortcuts: Object
```

Comment:
This one works. I was able to access the data while debugging and even the keystrokes worked. Should I be checking for anything else?